app/services/doc_to_pdf_converter.rb

```diff
@@ -1,13 +1,6 @@
 class DocToPdfConverter
   TMP_DIRECTORY = "./tmp/soffice".freeze
 
-  def self.convert_all
```

Comment:
There is a `lib/tasks/redacted_files.rake` file that was being used as a one-off task to call this that can be cleaned up.